app/javascript/elm/tests/LabelsInputTests.elm

```diff
@@ -47,7 +47,7 @@ all =
                     labels =
                         LabelsInput.fromList [ label ]
                 in
-                LabelsInput.view labels "description" "input-id" "placeholder" False Tooltip.profilesFilter
+                LabelsInput.view labels "description" "input-id" "placeholder" False Tooltip.profilesFilter "tooltip-icon"
```

Comment:
Why some are `tooltip-icon` and some `tooltip-icon.svg`? I guess it doesn't change anything, but it might be confusing.

Reply:
by chance, thx for pointing out